server/datastore/mysql/operating_system_vulnerabilities.go

```diff
@@ -214,16 +217,12 @@ SELECT DISTINCT
 FROM
 	software
 	LEFT JOIN software_cve ON software.id = software_cve.software_id
-	JOIN software_titles ON software_titles.id = software.title_id
-	JOIN host_software ON host_software.software_id = software.id
-	JOIN host_operating_system ON host_operating_system.host_id = host_software.host_id
-	JOIN operating_systems ON operating_systems.id = host_operating_system.os_id
-    JOIN software_host_counts ON software_host_counts.software_id = software.id
+	JOIN kernels ON kernels.software_id = software.id
+    JOIN software_host_counts ON software_host_counts.software_id = kernels.software_id
```

Comment:
Per my note on the DB migration, this won't get us the right number (and I missed that in previous review). Given that I only caught this the second time around, we definitely need an automated test with one kernel two OS versions to make sure we're returning the correct info.